tests/unit/orcid/fixtures/work.json

```diff
@@ -0,0 +1,151 @@
+                    "value": "Nucl.Phys."
+                }, 
+                "path": null, 
+                "short-description": null, 
+                "type": "JOURNAL_ARTICLE"
+            }
+        }, 
+        {
+            "work": {
+                "language-code": null,
+                "put-code": 912977,
+                "contributors": {
+                    "contributor": [
+                        {
+                            "contributor-orcid": null,
+                            "contributor-attributes": {
+                                "contributor-role": "AUTHOR",
+                                "contributor-sequence": "FIRST"
+                            },
+                            "credit-name": {
+                                "value": "Hu, Wayne"
+                            },
+                            "contributor-email": null
+                        }
+                    ]
+                },
+                "title": {
+                    "title": {
+                        "value": "CMB anisotropies: A Decadal survey"
+                    },
+                    "subtitle": null,
+                    "translated-title": null
+                },
+                "url": {
+                    "value": "http://localhost:5000/record/524480"
+                },
+                "country": null,
+                "last-modified-date": {
+                    "value": 1516716538439
+                },
+                "external-ids": {
+                    "external-id": [
+                        {
+                            "external-id-value": "astro-ph/0002520",
+                            "external-id-relationship": "SELF",
+                            "external-id-type": "arxiv",
+                            "external-id-url": {
+                                "value": "http://arxiv.org/abs/astro-ph/0002520"
+                            }
+                        }
+                    ]
+                },
+                "created-date": {
+                    "value": 1516715359801
+                },
+                "visibility": "PUBLIC",
+                "citation": null,
+                "source": {
+                    "source-orcid": null,
+                    "source-name": {
+                        "value": "INSPIRE-PROFILE-PUSH"
+                    },
+                    "source-client-id": {
+                        "path": "0000-0002-0970-6486",
+                        "host": "sandbox.orcid.org",
+                        "uri": "http://sandbox.orcid.org/client/0000-0002-0970-6486"
+                    }
+                },
+                "publication-date": null,
+                "journal-title": {
+                    "value": "4th RESCEU International Symposium on Birth and Evolution of the Universe"
+                },
+                "path": null,
+                "short-description": null,
+                "type": "CONFERENCE_PAPER"
+            }
+        }
+    ]
+}
```

Comment:
I would say it is always good to have a new line at the end of a file. It is also consistent with the rest of the the project.